GitHeart/API/API.swift

```diff
@@ -89,6 +89,8 @@ extension API: UsersListProvider {
     func users(searchTerm: String, page: Int, completion: @escaping ((Result<[User], Error>) -> Void)) {
         let query = searchTerm.isEmpty ? "followers:>1000" : searchTerm
         get(request: request(path: "/search/users", query: ["q": query, "page": String(page)]), completion: { result in
+          // Is it correct layer of abstraction to move completion into main queue?
```

Comment:
This is a public method that is used in a view layer or close to it. I'd expect it to call completion on main queue if otherwise it is not mentioned in the docs.